<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow_M_1</class>
 <widget class="QMainWindow" name="MainWindow_M_1">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget_M_1">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>-1</x>
      <y>-1</y>
      <width>1921</width>
      <height>1061</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="mainVLayout_M_1">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="sizeConstraint">
      <enum>QLayout::SetDefaultConstraint</enum>
     </property>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar_M_1">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1920</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar_M_1"/>
 </widget>
 <resources/>
 <connections/>
</ui>
